lefthook.yml

```
@@ -2,14 +2,14 @@ pre-commit:
   commands:
     biome_check:
       glob: 'packages/**/*.{js,ts,json}'
-      run: ./node_modules/.bin/biome check --write --no-errors-on-unmatched --files-ignore-unknown=true --colors=off {staged_files}
+      run: biome check --write --no-errors-on-unmatched --files-ignore-unknown=true --colors=off {staged_files}
```

can you please try adding `pnpm ` before the command, and check if that also works for you? 🙏🏽 

```suggestion
      run: pnpm biome check --write --no-errors-on-unmatched --files-ignore-unknown=true --colors=off {staged_files}
```

---

Still works with pnpm!
What does pnpm add in this case? :)

---

it ensures that the command works for all operating systems